google/cloud/common_options.h

```diff
@@ -66,6 +66,7 @@ struct UserAgentProductsOption {
  *
  * - rpc
  * - rpc-streams
+ * - http Only applicable when using REST transport.
```

Comment:
`http` already means something for `google/cloud/storage/`: it dumps every byte sent and received over the wire.  Why not just use `rpc` as before?